### ISSUE TITLE
Suggestion for a different type of fade (and improvement to "lined" fade type)

### ISSUE THREAD

**Issue by ZWORX52** [kerudion/chunksfadein#38]
**What do you want to change or add in the mod?**
I had an idea for a new type of fade. Summarized, they fade in a circle pattern out from the player. I know sub-block fade precision is possible, because the "Lined" fade type doesn't snap to blocks. You could also add another fade type which is the circle but with a little extra randomness, called something like "blob" or "infection." Finally, as an aside, is it possible to make not-fully-faded-in blocks or pixels truly transparent? Looking down at the "Lined" fade type, all you can see is the blue until the top blocks have "faded" in.

**Comment by kerudion** [comment]
Hi! Thanks a lot for the suggestions, they sound really interesting. About transparency, I tried it before and looks like it needs some modifications to work. Also, I think that it'll not look good and will allow looking through walls by setting fade time to maximum and holding F3 + A, which is straight up cheating.

**Comment by ZWORX52** [comment]
Yes, in hindsight I agree that transparency will essentially allow cheating in that regard (arguably like F3 + A in vanilla does). However, I do have another suggestion that could replace the transparency idea (I didn't include this at first because I thought it might be quite difficult): chunks could have a minimum time before fading if their neighbors just faded in (perhaps a configurable number of milliseconds). The idea is that with these animations, you always get that ripple effect even if the server is quicker to respond in some areas and it just looks cleaner.
